### ISSUE TITLE
Stray output when no implementations are specified

### ISSUE THREAD

**Issue by LourensVeen** [multiscale/ymmsl-python#10]
ymmsl-python emits a stray  `implementations: !!omap []` when saving a configuration without implementations specified, which is really annoying because those files are incompatible with older versions of the library.

- [x] Reproduce
- [x] Figure out why
- [x] Devise quick fix that people can use to work around it for now
- [x] Fix it properly

**Comment by DongweiYe** [comment]
The issue come up with using following version of library:
muscle3            0.4.0
PyYAML             5.3.1
ruamel.yaml        0.16.10
ruamel.yaml.clib   0.2.0
yatiml             0.7.0
ymmsl              0.11.0

The saving code is as follow:
```
def write_ymmsl_file(path,filename,config_data):
	with open(path+filename, 'w') as f:
		ymmsl.save(config_data, f)

config = ymmsl.Configuration(model, settings)
write_ymmsl_file(sample_path+'/','input_stage4.ymmsl',config)
```

If I remember right, a solution to this. I vaguely remember it was about a slightly different way to save the ymmsl files. I don't remember what exactly it was and also can't find it the slack history...

**Comment by DongweiYe** [comment]
```
config.resources = None
config.implementation = None
``` 
These two line of code helps to remove the redundant information.

**Comment by LourensVeen** [comment]
I'm getting a case of deja vu here as well :-). I think we discussed it before, and I think I suggested that work-around with assigning None, maybe already when making the `Configuration` object (i.e. `Configuration(model, settings, None, None)` instead of `Configuration(model, settings)`).

I've traced this to a mistake in the underlying YAtiML library, for which there's an issue https://github.com/yatiml/yatiml/issues/34. I'll fix that in the next release of YAtiML, then upgrade ymmsl-python to it with the next release.
